### PR TITLE
set up model_audit to handle PCA use case

### PR DESCRIPTION
**Pull request by driskerr**
the default setting for `model_audit` doesn't work for PCA models!

it does not have some of the other columns package is expecting like `loss`, `eval_loss`, `learning_rate`, so changing the settings here to only cover relevant columns.

### CODE REVIEW

**Review by rbjerrum — APPROVED** [review]

**Comment by driskerr** [comment]
thank you!!!!
